suite-native/passphrase/src/useFeatureRequestsPassphrase.tsx

```diff
@@ -0,0 +1,68 @@
+import { useCallback, useEffect } from 'react';
```

Comment:
What about simplifying this hook name to `usePassphrase` or something else shorter?

Reply:
I think this is more explanatory and won't be confused with `useHandleDeviceRequestsPassphrase`